src/frontend/src/pages/myfeeds.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Container, Box, Typography, TextField, Grid, Card, IconButton } from '@material-ui/core';
+import { AccountCircle, RssFeed, HelpOutline, Add } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
```

Comment:
Do we want to keep styles located in the react component? Is there a way to store this in the css file that should be located with the page to keep with the current format of the front end?

Reply:
Yes, it is technically possible to replace this instance of `makeStyles` with pure CSS, i.e.. 

```css
.MuiContainer-root {
  position: relative;
  top: 16px;
}

.MuiIconButton-root {
  padding: 3px 0 3px 0;
}
```

Relying more on [MaterialUI's styling solutions](https://material-ui.com/styles/basics/#hook-api) may become more useful in the future if/when we want to incorporate more MUI functionalities, such as customized [theming](https://material-ui.com/styles/advanced/#theming).

However, since I'm still pretty new to MUI, I'd like to defer to @miggs125 and @humphd regarding whether it would be better, in this case, to embrace or move away from MUI styling solutions.

Reply:
I agree - not sure if having each react component contain our materialUI solutions or if we should grab it from a separate page that would house all the material UI.

I do think we should keep anything CSS related outside of the functionality part. Perhaps we could import it and have a different file such as myFeedsCss.js? (or materials not sure what to call this file)

This would allow the CSS developers to quickly locate where the CSS is that would need to be changed, and not get them stuck in reading everything within a component?

Thoughts @humphd ?

Reply:
Depending on the outcome of #730, it looks like we might use MUI as a base.  If we do, then I think moving toward the MUI style approach is nice, because it takes less work to translate what their examples in the docs to to what we'll have.  Doing CSS-in-JS this way is kind of nice.

I think do whatever you want in this PR, and we'll change it to match #730.  I'll add a note about this topic there.